### PR TITLE
created universal pattern component

### PR DESCRIPTION
**Pull request by kirti75waytech**
@MaksymTaratuta55 @thecodeclimber I have resolved issue #102 under this PR. Please review it and let me know if any other modification is needed.
Thanks

### CODE REVIEW

File: src/components/elements/Pattern.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import PropTypes from "prop-types";
 
-const LeftCurveLines = ({ classes }) => (
+const Pattern = ({ classes }) => (
   <div className={classes.wrapper}>
-    <img src="/img/pattern__cta--left.svg" className={classes.img} />
+    <img src={`/img/${classes.name}.svg`} className={classes.img} />
```
Comment (MaksymTaratuta55) [inline src/components/elements/Pattern.jsx:5]: please, rename it from `${classes.name}.svg` to `imgName` and don't pass it in the `classes`

Comment (MaksymTaratuta55) [reply src/components/elements/Pattern.jsx:5]: Or you can create **Pattern.constants.js** with 
```
export const patternImages = {
    leftPattern: "/img/pattern__cta--left.svg",
    rightPattern: "/img/.....",
    .....
}
```
And use it in ``

**Review by MaksymTaratuta55 — CHANGES_REQUESTED** [review]

File: src/components/elements/Pattern.jsx (hunk 1 above)
Comment (kirti75waytech) [reply src/components/elements/Pattern.jsx:5]: Okay, sure @MaksymTaratuta55. I'll update the code as you have suggested.
Thanks

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

Can you please take care of these conflicts?  Thanks.

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 
@kirti75waytech 

I am not sure who is suppossed to resolve these conflicts.  Can you two please discuss and decide who should resolve them?  Thanks.

**Comment by kirti75waytech** [comment]
> @MaksymTaratuta55
> @kirti75waytech
> 
> I am not sure who is suppossed to resolve these conflicts. Can you two please discuss and decide who should resolve them? Thanks.

@thecodeclimber I will resolve conflicts for this PR.

**Comment by kirti75waytech** [comment]
@thecodeclimber I have resolved the conflicts for this PR. Please review the code and merge this PR.
Thanks

**Review by MaksymTaratuta55 — APPROVED** [review]

**Comment by thecodeclimber** [comment]
@kirti75waytech 

I made some changes to the `Pattern` component.  Here is a video showing what I did:  https://www.loom.com/share/8e025225dffc417596d34a284f51aca5

I am now going to merge the branch into `develop--clone`.

**Pull request merged**